kaldb/src/main/java/com/slack/kaldb/preprocessor/PreprocessorService.java

```diff
@@ -261,32 +252,50 @@ protected static List<Integer> getActivePartitionList(DatasetMetadata datasetMet
         .collect(Collectors.toUnmodifiableList());
   }
 
+  public static List<DatasetMetadata> sortDatasetsOnThroughput(
```

Comment:
Maybe a comment here, or where it's called explaining why we sort on throughput.